nixos/modules/services/x11/xautolock.nix

```diff
@@ -45,28 +55,80 @@ in
         };
 
         notifier = mkOption {
-          default = "notify-send 'Locking in 10 seconds'";
-          type = types.string;
+          default = null;
+          example = "notify-send 'Locking in 10 seconds'";
```

Comment:
Would this example actually work if `libnotify` is not installed? Should this not be `"${pkgs.libnotify}/bin/notify-send"`.

Reply:
true, but the example is just for documentation purposes and nothing that actually influences the system, right?

Reply:
No it actually won't, but I'm not sure what example should be used there, so I kept the previous one. I'd prefix it as you did if this example with notify-send is acceptable, even though it does not notify me on my own system like that. I'm using i3wm and here it probably would be more fitting to use something like `i3-nagbar -m 'locking soon'`.
I'm open for suggestions :)

Reply:
I just got it working. I need to have `notify-osd` running and then `notify-send` works, which means anyone using `notify-send` has to enable/run the daemon manually. I think this is not for the xautolock module to handle, right?
If so, I'd just prefix the command in the example :)

Reply:
Using the following value should do the trick:

```
example = literalExample ''"${pkgs.libnotify}/bin/notify-send \"Locking in 10 seconds\"''';
```